core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.ts

```diff
@@ -1299,7 +1295,7 @@ export class ConversationSkinComponent {
           }
         }
         if (
```

Comment:
Add a line break above this, because it's not actually connected to the previous if clause (correct?)

Reply:
Added